aux/flatpak/com.github.akiraux.akira.json

```diff
@@ -23,21 +23,7 @@
     "--filesystem=xdg-run/gvfs",
     "--filesystem=xdg-run/gvfsd"
```

Comment:
We should drop one of the two manifests.

Reply:
which one would you drop?